.tekton/collector-slim-pull-request.yaml

```diff
@@ -213,12 +217,27 @@ spec:
       - name: basic-auth
         workspace: git-auth
 
+    - name: download-external-content
```

Comment:
How about calling this `fetch-external-content`? This way, the name is still ok-ish for `collector-slim` which does not need to _download_ things from network and only injects the activation key (which is in a way external).

Reply:
That's a copy&paste artifact. I renamed the step to `prepare-rhel-rpm-subscriptions` in https://github.com/stackrox/collector/pull/1573/commits/23d4abc2834f524c7d4eed81dd9e5e67d7be4d70